<map id="Mezzanine::NetworkURLException" name="Mezzanine::NetworkURLException">
<area shape="rect" id="node2" href="$classMezzanine_1_1NetworkException.xhtml" title="Thrown when something unknown causes network IO to fail." alt="" coords="19,237,229,267"/>
<area shape="rect" id="node4" href="$classMezzanine_1_1IOException.xhtml" title="Thrown when there was an issue with IO but very little was known about it." alt="" coords="37,160,211,189"/>
<area shape="rect" id="node6" href="$classMezzanine_1_1Exception.xhtml" title="This is the exception thrown by most Mezzanine system that can throw exceptions." alt="" coords="44,83,204,112"/>
<area shape="rect" id="node8" title="STL class." alt="" coords="68,5,180,35"/>
</map>
